docs/guide/web-simulation.md

```diff
@@ -54,13 +54,18 @@ Then, start a session and simulation servers:
 cd $WEBOTS_HOME/resources/web/server
 ./server.sh start default
 ```
+
+If you have installed Webots with the `.deb` or the `.snap` package, you should:
+ - Copy the server folder in a place where they can be launched without `sudo` and run the scripts from there.
+ - Open the `default.json file` located in `server/config/simulation/` with you ide/text editor and add the following line between the `{}`: "webotsHome": "/usr/local/webots". This line is used to tell the server where Webots is located
```

Comment:
There is no `default.json` file in the `server/config/simulation` folder.

Reply:
On R2022a there is